internal/compliance/policy_engine/src/policy.py

```diff
@@ -76,6 +78,12 @@ def __init__(self, policies: List[Dict[str, Any]]) -> None:
         self._policies_by_type: Dict[str, List[Policy]] = collections.defaultdict(list)
         self._global_policies: List[Policy] = []  # List of policies that apply to all log types
 
+        # Setup panther helper functions
+        helpers_spec = import_util.spec_from_file_location(helpers.__name__, helpers.__file__)
+        helpers_mod = import_util.module_from_spec(helpers_spec)
+        sys.modules['panther_helpers'] = helpers_mod
+        helpers_spec.loader.exec_module(helpers_mod)  # type: ignore
```

Comment:
We have a static method for this very purpose:

```suggestion
sys.modules['panther_helpers'] = Policy.import_module(helpers.__name__, helpers.__file__)
```

Also consider naming this library just `panther`. It reads a little cleaner in the policy:

```python
import panther

def policy(resource):
    panther.resource_lookup(...)
```